examples/nextjs-14-sensitive-info/README.md

```diff
@@ -0,0 +1,38 @@
+<a href="https://arcjet.com" target="_arcjet-home">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://arcjet.com/logo/arcjet-dark-lockup-voyage-horizontal.svg">
+    <img src="https://arcjet.com/logo/arcjet-light-lockup-voyage-horizontal.svg" alt="Arcjet Logo" height="128" width="auto">
+  </picture>
+</a>
+
+# Arcjet Block Sensitive Info with Next.js 14 using the App Router
+
+This example shows how to use Arcjet with a Next.js [route
+handler](https://nextjs.org/docs/app/building-your-application/routing/route-handlers).
+
+## How to use
+
+1. From the root of the project, install the SDK dependencies.
+
+   ```bash
+   npm ci
+   ```
+
+2. Enter this directory and install the example's dependencies.
+
+   ```bash
+   cd examples/nextjs-14-app-dir-rl
+   npm ci
+   ```
+
+3. Rename `.env.local.example` to `.env.local` and add your Arcjet key.
+
+4. Start the dev server.
+
+   ```bash
+   npm run dev
+   ```
+
+5. Curl `http://localhost:3000/api/arcjet` with some data
+   `curl http://localhost:3000/api/arcjet --data "my email is test@example.com"`
```

Comment:
This example will be allowed because you allow email. Should we add a sample that will be blocked?